config.toml

```diff
@@ -41,14 +41,9 @@ link = "https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.cs
 # Flowbite
 link = "https://cdnjs.cloudflare.com/ajax/libs/flowbite/1.6.5/flowbite.min.css"
 
-[[params.plugins.css]]
-# HighlightJS
```

Comment:
A `HighlightJS` script was commented out, so I decided to check and see if we were using the plugin. After checking blog posts with code and the data playground pages, I was able to confirm that we weren't using this plugin.